doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -211,6 +211,10 @@ nRF Desktop
   * Changed the term *flash memory* to *non-volatile memory* for generalization purposes.
   * The :ref:`nrf_desktop_usb_state` to use the :c:func:`usb_hid_set_proto_code` function to set the HID Boot Interface protocol code.
     The ``CONFIG_USB_HID_PROTOCOL_CODE`` Kconfig option is deprecated and a dedicated API needs to be used instead.
+  * The :ref:`nrf_desktop_usb_state` to use :c:func:`usb_hid_set_proto_code` function to set the HID Boot Interface protocol code.
+    The ``CONFIG_USB_HID_BOOT_PROTOCOL`` Kconfig option was removed and dedicated API needs to be used instead.
+  * Disabled MCUboot's logs over RTT (:kconfig:option:`CONFIG_LOG_BACKEND_RTT` and :kconfig:option:`CONFIG_USE_SEGGER_RTT`) on ``nrf52840dk_nrf52840`` in :file:`prj_mcuboot_qspi.conf` configuration to reduce MCUboot memory footprint and avoid flash overflows.
+    Explicitly enabled the UART log backend (:kconfig:option:`CONFIG_LOG_BACKEND_UART`) together with its dependencies in the configuration file to ensure log visibility.
```

Comment:
@MarekPieta , can you check these lines for accuracy?

Reply:
The first bullet point here seems duplicated. We should remove it. Apart from it looks good :)
@doer-nordic can we remove it, please?

Reply:
Will fix